### PR TITLE
Remove unused AreaMapView

### PR DESCRIPTION
**Pull request by UsamaEquinorAFK**

### CODE REVIEW

File: frontend/src/components/Pages/RobotPage/AreaMapView.tsx [hunk 1]
```diff
@@ -1,174 +0,0 @@
...
-        const image = new Image()
-        image.src = url
-        await image.decode()
-        return image
-    }
-
-    useEffect(() => {
-        setIsLoading(true)
-        setImageObjectURL(undefined)
-        BackendAPICaller.getAreasMapMetadata(area.id)
```
Comment (Eddasol) [inline frontend/src/components/Pages/RobotPage/AreaMapView.tsx:77]: Not used elsewhere. Can also be removed

File: frontend/src/components/Pages/RobotPage/AreaMapView.tsx [hunk 2]
```diff
@@ -1,174 +0,0 @@
-import { CircularProgress, Typography } from '@equinor/eds-core-react'
-import { MouseEvent, useCallback, useEffect, useState } from 'react'
-import styled from 'styled-components'
-import NoMap from 'mediaAssets/NoMap.png'
-import { placeRobotInMap, inverseCalculatePixelPosition } from 'utils/MapMarkers'
```
Comment (Eddasol) [inline frontend/src/components/Pages/RobotPage/AreaMapView.tsx:5]: inverseCalculatePixelPosition is only used here. Implementation can also be deleted

File: frontend/src/components/Pages/RobotPage/AreaMapView.tsx (hunk 1 above)
Comment (UsamaEquinorAFK) [reply frontend/src/components/Pages/RobotPage/AreaMapView.tsx:77]: Done !

File: frontend/src/components/Pages/RobotPage/AreaMapView.tsx (hunk 2 above)
Comment (UsamaEquinorAFK) [reply frontend/src/components/Pages/RobotPage/AreaMapView.tsx:5]: Done.

File: frontend/src/api/ApiCaller.tsx [hunk 3]
```diff
@@ -440,15 +440,6 @@ export class BackendAPICaller {
         return result.content
     }
 
-    static async getAreasMapMetadata(id: string): Promise<MapMetadata> {
-        const path: string = 'areas/' + id + '/map-metadata'
```
Comment (Eddasol) [inline frontend/src/api/ApiCaller.tsx:444]: Is the backend endpoint in use anymore? If not that can probably be removed as well

Comment (UsamaEquinorAFK) [reply frontend/src/api/ApiCaller.tsx:444]: Will investigate.

Comment (UsamaEquinorAFK) [reply frontend/src/api/ApiCaller.tsx:444]: Done.

**Review by andchiind — APPROVED** [review]
LGTM

**Pull request merged**